packages/contracts-bedrock/src/libraries/Constants.sol

```diff
@@ -50,4 +52,14 @@ library Constants {
         });
         return config;
     }
+
+    function DEFAULT_STARTING_ANCHOR_ROOTS() internal pure returns (IAnchorStateRegistry.StartingAnchorRoot[] memory) {
```

Comment:
Hmm one thing that I don't love is that this lives in `src`, but is not used by any contracts in `src/`

Reply:
[df735c7](https://github.com/ethereum-optimism/optimism/pull/12320/commits/df735c7b79408f7b8b37aac903bdbec332487ef0)